src/opl/generator.h

```diff
@@ -142,12 +151,22 @@ class Generator
         NotesManager();
         ~NotesManager();
         void allocateChannels(int count);
-        uint8_t noteOn(int note);
+        uint8_t noteOn(int note, bool *replace = nullptr);
         int8_t  noteOff(int note);
+        void    channelOff(int ch);
+        int8_t  findNoteOffChannel(int note);
+        void hold(int ch, bool h);
         void clearNotes();
+        const Note &channel(int ch) const
+            { return channels.at(ch); }
+        const int channelCount() const
```

Comment:
GCC 6 says me `const` is junk here.
So, you are free to define
```
int channelCount() const
```
as returning type is not a pointer or even not a reference :wink: 
I have fixed it on my side, and if everything is fine, I'll push my state to master

Reply:
I have no idea why I have even written that. The fingers act before brain thinks sometimes.